bokeh/io/webdriver.py

```diff
@@ -65,6 +65,7 @@ def create_chromium_webdriver() -> WebDriver:
     options.add_argument("--hide-scrollbars")
     options.add_argument("--force-device-scale-factor=1")
     options.add_argument("--force-color-profile=srgb")
+    options['loggingPrefs'] = {'browser': 'ALL'}
```

Comment:
This won't have an effect, because in tests there is a pytest plugin that manages webdrivers (`bokeh._testing.plugins.selenium`).